AsyncDisplayKit/Layout/ASAbsoluteLayoutSpec.mm

```diff
@@ -109,13 +121,36 @@ - (BOOL)supportsUpwardPropagation
 
 @end
 
-@implementation ASAbsoluteLayoutSpec (Debugging)
+#pragma mark - Debugging
 
-#pragma mark - ASLayoutElementAsciiArtProtocol
+@implementation ASAbsoluteLayoutSpec (Debugging)
 
 - (NSString *)debugBoxString
 {
   return [ASLayoutSpec asciiArtStringForChildren:self.children parentName:[self asciiArtName]];
 }
 
 @end
+
+
+#pragma mark - ASStaticLayoutSpec
+
+@implementation ASStaticLayoutSpec : ASAbsoluteLayoutSpec
+
++ (instancetype)staticLayoutSpecWithChildren:(NSArray<id<ASLayoutElement>> *)children
+{
+  return [self absoluteLayoutSpecWithChildren:children];
+}
+
+- (instancetype)initWithChildren:(NSArray *)children
+{
+  if (!(self = [super initWithChildren:children])) {
```

Comment:
Should call `initWithSizing:children:` with `ASAbsoluteLayoutSpecSizingSizeToFit`